railties/test/application/rake/multi_dbs_test.rb

```diff
@@ -413,6 +403,7 @@ def generate_models_for_animals
       end
 
       test "db:migrate and db:structure:dump and db:structure:load works on all databases" do
```

Comment:
Is it? 
It actually internally runs `db:migrate`, `db:structure:dump` and `db:structure:load`.

Reply:
Since we're not going to delete this test and are just going to delete the `structure:dump` and `structure:load` methods when the deprecation is complete, this test name will not match the public API anymore so it should be "test "db:migrate and db:schema:dump and db:schema:load works on all databases with sql option". Then the test method `db_migrate_and_schema_dump_and_load ` should be changed to pass `sql` instead of `structure`.

Reply:
Updated with your suggestion.